lib/grape/dsl/inside_route.rb

```diff
@@ -191,8 +191,12 @@ def body(value = nil)
       #
       #   GET /file # => "contents of file"
       def file(value = nil)
-        if value
-          @file = Grape::Util::FileResponse.new(value)
+        if value.is_a?(String)
+          file_body = Grape::ServeFile::FileBody.new(value)
+          @file = Grape::ServeFile::FileResponse.new(file_body)
+        elsif !value.is_a?(NilClass)
+          warn '[DEPRECATION] Argument as FileStreamer-like object will be deprecated. Use path to file instead'
```

Comment:
Thanks. If you're rubbing the bext version the old code is already deprecated but it does work with a warning. I think present tense is correct :) thanks for making the changes!